Index: src/main/java/com/taxi_booking/taxi_booking_backend/repository/PaymentRepository.java
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/repository/PaymentRepository.java b/src/main/java/com/taxi_booking/taxi_booking_backend/repository/PaymentRepository.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/repository/PaymentRepository.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,7 +0,0 @@
-package com.taxi_booking.taxi_booking_backend.repository;
-
-import com.taxi_booking.taxi_booking_backend.entity.Payment;
-import org.springframework.data.jpa.repository.JpaRepository;
-
-public interface PaymentRepository extends JpaRepository<Payment, Long> {
-}
Index: src/main/java/com/taxi_booking/taxi_booking_backend/entity/CashPayment.java
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/CashPayment.java b/src/main/java/com/taxi_booking/taxi_booking_backend/entity/CashPayment.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/CashPayment.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,20 +0,0 @@
-package com.taxi_booking.taxi_booking_backend.entity;
-
-import jakarta.persistence.Entity;
-
-@Entity
-public class CashPayment extends Payment {
-
-    public CashPayment() {
-    }
-
-    public CashPayment(Double amount, Long bookingId) {
-        super(amount, bookingId);
-    }
-
-    @Override
-    public String processPayment() {
-        setPaymentStatus(PaymentStatus.PAID);
-        return "Cash collected by driver successfully.";
-    }
-}
\ No newline at end of file
Index: src/main/java/com/taxi_booking/taxi_booking_backend/TaxiBookingBackendApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/TaxiBookingBackendApplication.java b/src/main/java/com/taxi_booking/taxi_booking_backend/TaxiBookingBackendApplication.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/TaxiBookingBackendApplication.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,17 +0,0 @@
-package com.taxi_booking.taxi_booking_backend;
-
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-
-@SpringBootApplication
-public class TaxiBookingBackendApplication {
-
-	public static void main(String[] args) {
-		SpringApplication.run(TaxiBookingBackendApplication.class, args);
-		System.out.println("Hellow");
-		System.out.println("hello");
-		System.out.println("hyy");
-		System.out.println("username");
-	}
-
-}
Index: src/main/java/com/taxi_booking/taxi_booking_backend/entity/PaymentStatus.java
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/PaymentStatus.java b/src/main/java/com/taxi_booking/taxi_booking_backend/entity/PaymentStatus.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/PaymentStatus.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,7 +0,0 @@
-package com.taxi_booking.taxi_booking_backend.entity;
-
-public class PaymentStatus {
-    PENDING,
-    PAID,
-    FAILED
-}
Index: src/main/java/com/taxi_booking/taxi_booking_backend/dto/PaymentInvoiceDto.java
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/dto/PaymentInvoiceDto.java b/src/main/java/com/taxi_booking/taxi_booking_backend/dto/PaymentInvoiceDto.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/dto/PaymentInvoiceDto.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,41 +0,0 @@
-package com.taxi_booking.taxi_booking_backend.dto;
-
-public class PaymentInvoiceDto {
-
-    private Double amount;
-    private Long bookingId;
-    private String paymentType;
-    private String cardNumber;
-
-    public Double getAmount() {
-        return amount;
-    }
-
-    public void setAmount(Double amount) {
-        this.amount = amount;
-    }
-
-    public Long getBookingId() {
-        return bookingId;
-    }
-
-    public void setBookingId(Long bookingId) {
-        this.bookingId = bookingId;
-    }
-
-    public String getPaymentType() {
-        return paymentType;
-    }
-
-    public void setPaymentType(String paymentType) {
-        this.paymentType = paymentType;
-    }
-
-    public String getCardNumber() {
-        return cardNumber;
-    }
-
-    public void setCardNumber(String cardNumber) {
-        this.cardNumber = cardNumber;
-    }
-}
\ No newline at end of file
Index: src/main/java/com/taxi_booking/taxi_booking_backend/entity/CardPayment.java
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/CardPayment.java b/src/main/java/com/taxi_booking/taxi_booking_backend/entity/CardPayment.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/CardPayment.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,37 +0,0 @@
-package com.taxi_booking.taxi_booking_backend.entity;
-
-import jakarta.persistence.Entity;
-
-@Entity
-public class CardPayment extends Payment {
-
-    private String cardNumber;
-
-    public CardPayment() {
-    }
-
-    public CardPayment(Double amount, Long bookingId, String cardNumber) {
-        super(amount, bookingId);
-        this.cardNumber = cardNumber;
-    }
-
-    @Override
-    public String processPayment() {
-
-        if(cardNumber.length() >= 4) {
-            setPaymentStatus(PaymentStatus.PAID);
-            return "Connected to VISA API. Payment Successful.";
-        }
-
-        setPaymentStatus(PaymentStatus.FAILED);
-        return "Payment Failed.";
-    }
-
-    public String getCardNumber() {
-        return cardNumber;
-    }
-
-    public void setCardNumber(String cardNumber) {
-        this.cardNumber = cardNumber;
-    }
-}
\ No newline at end of file
Index: src/main/java/com/taxi_booking/taxi_booking_backend/controller/PaymentController.java
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/controller/PaymentController.java b/src/main/java/com/taxi_booking/taxi_booking_backend/controller/PaymentController.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/controller/PaymentController.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,41 +0,0 @@
-package com.taxi_booking.taxi_booking_backend.controller;
-
-import com.taxi_booking.taxi_booking_backend.dto.PaymentInvoiceDto;
-import com.taxi_booking.taxi_booking_backend.entity.Payment;
-import com.taxi_booking.taxi_booking_backend.entity.PaymentStatus;
-import com.taxi_booking.taxi_booking_backend.service.PaymentService;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
-
-import java.util.List;
-
-@RestController
-@RequestMapping("/payments")
-@CrossOrigin("*")
-public class PaymentController {
-
-    @Autowired
-    private PaymentService paymentService;
-
-    // CREATE
-    @PostMapping("/invoice")
-    public String generateInvoice(@RequestBody PaymentInvoiceDto dto) {
-        return paymentService.generateInvoice(dto);
-    }
-
-    // READ
-    @GetMapping
-    public List<Payment> getAllPayments() {
-        return paymentService.viewBillingHistory();
-    }
-
-    // UPDATE
-    @PutMapping("/{id}")
-    public Payment updateStatus(
-            @PathVariable Long id,
-            @RequestParam PaymentStatus status) {
-
-        return paymentService.updatePaymentStatus(id, status);
-    }
-}
\ No newline at end of file
Index: src/main/java/com/taxi_booking/taxi_booking_backend/entity/Payment.java
===================================================================
diff --git a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/Payment.java b/src/main/java/com/taxi_booking/taxi_booking_backend/entity/Payment.java
deleted file mode 100644
--- a/src/main/java/com/taxi_booking/taxi_booking_backend/entity/Payment.java	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
+++ /dev/null	(revision e13ce0193fd0251f2d5a3f1bd0ebc30adaa6d171)
@@ -1,69 +0,0 @@
-package com.taxi_booking.taxi_booking_backend.entity;
-
-import jakarta.persistence.*;
-import java.time.LocalDateTime;
-
-@Entity
-@Inheritance(strategy = InheritanceType.JOINED)
-public abstract class Payment {
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long paymentId;
-
-    private Double amount;
-
-    private LocalDateTime paymentDate;
-
-    @Enumerated(EnumType.STRING)
-    private PaymentStatus paymentStatus;
-
-    private Long bookingId;
-
-    // Constructors
-    public Payment() {
-    }
-
-    public Payment(Double amount, Long bookingId) {
-        this.amount = amount;
-        this.bookingId = bookingId;
-        this.paymentDate = LocalDateTime.now();
-        this.paymentStatus = PaymentStatus.PENDING;
-    }
-
-    // Abstract Method (Polymorphism)
-    public abstract String processPayment();
-
-    // Getters and Setters
-    public Long getPaymentId() {
-        return paymentId;
-    }
-
-    public Double getAmount() {
-        return amount;
-    }
-
-    public void setAmount(Double amount) {
-        this.amount = amount;
-    }
-
-    public LocalDateTime getPaymentDate() {
-        return paymentDate;
-    }
-
-    public PaymentStatus getPaymentStatus() {
-        return paymentStatus;
-    }
-
-    public void setPaymentStatus(PaymentStatus paymentStatus) {
-        this.paymentStatus = paymentStatus;
-    }
-
-    public Long getBookingId() {
-        return bookingId;
-    }
-
-    public void setBookingId(Long bookingId) {
-        this.bookingId = bookingId;
-    }
-}
\ No newline at end of file
